server/odc-service/src/main/java/com/oceanbase/odc/service/common/DruidMonitor.java

```diff
@@ -0,0 +1,646 @@
+        List<String> msg = methodStatVOS.stream().filter(
+                t -> t.computeAvgJdbcExecuteTime() > getLongMonitorConfig(DruidMonitorConfig.METHOD_JDBC_EXECUTE_TIME))
+                .map(
+                        DruidMethodStatVO::buildTooLongJdbcAvgExecuteTimeMsg)
+                .collect(
+                        Collectors.toList());
+        if (!msg.isEmpty()) {
+            AlarmUtils.alarm(METHOD_TOO_LONG_EXECUTE_TIME, "method too long avg jdbc execute time" + msg);
+        }
+    }
+
+    private void monitorMethodTooMuchJdbcExecute(List<DruidMethodStatVO> methodStatVOS) {
+        List<String> msg = methodStatVOS.stream()
+                .filter(t -> t
+                        .computeAvgJdbcExecuteCount() > getLongMonitorConfig(DruidMonitorConfig.METHOD_JDBC_COUNT))
+                .map(
+                        DruidMethodStatVO::buildTooMuchJdbcExecuteCountMsg)
+                .collect(
+                        Collectors.toList());
+        if (!msg.isEmpty()) {
+            AlarmUtils.alarm(METHOD_TOO_MUCH_JDBC_EXECUTE_COUNT, "method too much jdbc execute count" + msg);
+        }
+    }
+
+    @Data
+    public static final class DruidMethodStatConstants {
+
+        public static final String CLASS = "Class";
+        public static final String METHOD = "Method";
+        public static final String RUNNING_COUNT = "RunningCount";
+        public static final String CONCURRENT_MAX = "ConcurrentMax";
+        public static final String EXECUTE_COUNT = "ExecuteCount";
+        public static final String EXECUTE_ERROR_COUNT = "ExecuteErrorCount";
+        public static final String EXECUTE_TIME_MILLIS = "ExecuteTimeMillis";
+        public static final String JDBC_COMMIT_COUNT = "JdbcCommitCount";
+        public static final String JDBC_READ_ONLY_COUNT = "JdbcReadOnlyCount";
+        public static final String JDBC_ROLLBACK_COUNT = "JdbcRollbackCount";
+        public static final String JDBC_POOL_CONNECTION_OPEN_COUNT = "JdbcPoolConnectionOpenCount";
+        public static final String JDBC_POOL_CONNECTION_CLOSE_COUNT = "JdbcPoolConnectionCloseCount";
+        public static final String JDBC_RESULT_SET_OPEN_COUNT = "JdbcResultSetOpenCount";
+        public static final String JDBC_RESULT_SET_CLOSE_COUNT = "JdbcResultSetCloseCount";
+        public static final String JDBC_EXECUTE_COUNT = "JdbcExecuteCount";
+        public static final String JDBC_EXECUTE_ERROR_COUNT = "JdbcExecuteErrorCount";
+        public static final String JDBC_EXECUTE_TIME_MILLIS = "JdbcExecuteTimeMillis";
+        public static final String JDBC_FETCH_ROW_COUNT = "JdbcFetchRowCount";
+        public static final String JDBC_UPDATE_COUNT = "JdbcUpdateCount";
+        public static final String LAST_ERROR = "LastError";
+        public static final String LAST_ERROR_TIME = "LastErrorTime";
+        public static final String HISTOGRAM = "Histogram";
+
+        private DruidMethodStatConstants() {}
```

Comment:
why should we declare this constructor?

Reply:
Utility classes, which are collections of static members, are not meant to be instantiated.